app/views/aid_applications/duplicates/show.html.erb

```diff
@@ -8,27 +8,50 @@
     <div class="form-card__content">
       <p><%= t('.body') %></p>
     </div>
+    <div class="data-table-container">
+      <table class="data-table js-sticky-header">
```

Comment:
Just wanted to note that I don't think a table is a good way (currently it's inaccessable because of empty headers) to present this information but I don't think Honeycrisp has something like an HTML description list, e.g. https://getbootstrap.com/docs/4.0/content/typography/#description-list-alignment

Reply:
Agreed. i'll make an issue in the honeycrisp repo